sdk/formrecognizer/azure-ai-formrecognizer/pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.5.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.6.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
`version_client.txt` should also be updated to include the unreleased dependency.

Reply:
The unreleased dependency for azure-core is already in [version_client.txt](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L65). The failures in the pipelines are due to test failures in formrecognizer and checkstyle failures in core due to changes in this PR.